src/main/java/org/jenkinsci/plugins/ghprb/GhprbPullRequestMerge.java

```diff
@@ -37,11 +37,11 @@
 
     private transient PrintStream logger;
 
-    private final Boolean onlyAdminsMerge;
-    private final Boolean disallowOwnCode;
+    private final boolean onlyAdminsMerge;
```

Comment:
Using `boolean` prevents it from the possibility of being null when the get methods are called.  And even if Jenkins is assigning these via reflection upon reboot, I can't imagine their code would be applying explicit nulls to primitives.  I'll change it back if you want but it feels like using the `Boolean` object is just asking for more trouble and requiring explicit value checks.